packages/node_modules/@ciscospark/widget-space/src/actions.js

```diff
@@ -1,4 +1,22 @@
+import {isUuid} from '@ciscospark/react-component-utils';
+
+export const FETCH_SPACE_ERROR = `widget-space/FETCH_SPACE_ERROR`;
```

Comment:
I'm not sure what the module would do. Each one of these errors will probably be a unique case. Unless you meant moving the fetch space details into its own module.